testapi.pm

```diff
@@ -525,19 +556,20 @@ sub wait_serial {
 
     # wait for a message to appear on serial output
     my $regexp           = shift;
-    my $timeout          = shift || 90;    # seconds
-    my $expect_not_found = shift || 0;     # expected can not found the term in serial output
+    my $timeout          = shift || 90;                                      # seconds
+    my $expect_not_found = shift || 0;                                       # expected can not found the term in serial output
```

Comment:
that comment moved a bit far away from the code. I suggest to either add a blank line after this line and fix indendation or put the comment above each code line.